atlp-rwanda/e-commerce-serge-30-bn
# #187364839 Set up PostgreSQL (Supabase) as DB Engine with Sequelize ORM

**#### What does this PR do?**

This PR integrates PostgreSQL with our Express application to provide database persistence for user data.

**#### Description of Task to be completed?**

- Configures Sequelize ORM to connect Express with a PostgreSQL database.
- Sets up initial migrations and models for managing user data.
- Implements CRUD operations for user entities in the Express routes for testing.

**#### How should this be manually tested?**

- Start the Express server and ensure it successfully connects to the PostgreSQL database.
- Manually test each CRUD operation (create, read, update, delete) for user entities using tools like Postman or Thunderclient
![Screenshot from 2024-04-09 15-21-46](https://github.com/atlp-rwanda/e-commerce-serge-30-bn/assets/126393904/add97917-e780-45b7-9a2b-2a728da4a3ef)

**#### What are the relevant pivotal tracker/Trello stories?**

#187364838
